apps/web/pages/availability/[schedule].tsx

```diff
@@ -83,11 +101,88 @@ const DateOverride = ({ workingHours }: { workingHours: WorkingHours[] }) => {
   );
 };
 
+const DeleteDialogButton = ({
+  disabled,
+  scheduleId,
+  buttonClassName,
+  onDeleteConfirmed,
+}: {
+  disabled?: boolean;
+  onDeleteConfirmed?: () => void;
+  buttonClassName: string;
+  scheduleId: number;
+}) => {
+  const { t } = useLocale();
+  const router = useRouter();
+  const utils = trpc.useUtils();
+  const { isPending, mutate } = trpc.viewer.availability.schedule.delete.useMutation({
+    onError: (err) => {
+      if (err instanceof HttpError) {
+        const message = `${err.statusCode}: ${err.message}`;
+        showToast(message, "error");
+      }
```

Comment:
err could also be of type TRPCClientError. there is no toast message in case of Trpc error.

Reply:
Made some fixes